packages/cxl-ui/scss/cxl-app-layout.scss

```diff
@@ -189,6 +188,10 @@ main {
     display: flex;
     flex-direction: column;
   }
+
+  main {
+    z-index: 1;
+  }
```

Comment:
What's this change for?

Reply:
It fixes a stacking context issue when using the reversed theme (2c-l). The menu collapse button would appear above the shadow of the main content container.

Reply:
OK I confirmed the issue :+1:  and significantly clarified the code comment